### ISSUE TITLE
Replying to some mails leave To-field empty

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#527]
**Originally by _AmigaSurfer@t-online.de_ on 2014-02-08 10:36:14 +0100**

___
## Summary
At some mails the To-field leaves empty when replying

## Steps to reproduce
 1. Open the mail
 2. Press reply

## Expected results
To-field with sender

## Actual results
To-field is empty

## Notes
First seen at Nightly Build from 18.01.2014.
Still happens at Nightly Build from 29.01.2014.
Because this happens only on less mails, maybe already happens in an earlier version.

**Comment by jens-maus** [comment]
**Originally by _AmigaSurfer@t-online.de_ on 2014-02-08 10:37:15 +0100**

___
**Attachment added:** [`Reply-To Testmails.lha`](https://yam.ch/attachments/527/Reply-To%20Testmails.lha) (6.6 KiB)
Mails for testing Reply

**Comment by tboeckel** [comment]
**Originally on 2014-02-08 12:23:59 +0100**

___
I just tried both mails here on AmigaOS3/WinUAE and replying to them puts the correct address in the To field.

Did you try your modified test mails yourself?

**Comment by jens-maus** [comment]
**Originally by _AmigaSurfer@t-online.de_ on 2014-02-09 19:46:43 +0100**

___
The modified test mails puts the correct address in the To field here, too.
Maybe the bug associated with the mailinglist setting of the folder and the address fields of the mail. See PM.

**Comment by tboeckel** [comment]
**Originally on 2014-02-24 10:21:36 +0100**

___
This is the private mail of Frank:
--------------------------------------
Bei den geänderten Beispielmails wird bei mir jetzt auch der Absender
als neuer Empfänger übernommen. Das gleiche, wenn ich die orignial
Mails in einem anderen Ordner ohne Mailinglistenunterstützung
importiere. Hängt vielleicht mit der Mailinglistenunterstützung
zusammen. Ich habe mich eben auch gewundert, weshalb ich für einen
normalen Postfach-Ordner die Mailinglistenunterstützung aktiviert
habe. Aber das ist wohl die einzigste Möglichkeit, Mails aus
verschiedenen Ordnern heraus automatisch mit den entsprechenden
Absendern und Signaturen zu versehen.
--------------------------------------

Basically it seems that this issue is triggered by certain folder settings, in particular the mailling list support settings.

**Comment by tboeckel** [comment]
**Originally on 2014-02-24 10:23:46 +0100**

___
Replying to AmigaSurfer:
> The modified test mails puts the correct address in the To field here, too.
> Maybe the bug associated with the mailinglist setting of the folder and the address fields of the mail. See PM.

Frank, is it ok for you when I attach your screenshot of the folder settings to this ticket?

**Comment by jens-maus** [comment]
**Originally by _AmigaSurfer@t-online.de_ on 2014-02-24 19:13:02 +0100**

___
Yes, it's ok.

**Comment by tboeckel** [comment]
**Originally on 2014-02-24 21:41:14 +0100**

___
**Attachment added:** `pic00.png` (24.3 KiB)
![pic00.png](https://yam.ch/attachments/527/pic00.png)
Screenshot of the folder settings

**Comment by tboeckel** [comment]
**Originally on 2014-02-24 21:45:53 +0100**

___
Now that I took another look at the screenshot I noticed that you are using the wrong receiver pattern. This is not your own address, but the address you must send posts to the mailling list to to let them appear on the list.

The "Auto detect" button will let YAM scan some mails in the folder to extract a common address pattern.

Personalized mails like the example mails are no mails from a mailing list and should not be treated as such.

**Comment by jens-maus** [comment]
**Originally by _AmigaSurfer@t-online.de_ on 2014-02-25 21:16:01 +0100**

___
This is not a mailing list folder. It's a folder for personal mails from a specific account. I'm misusing the mailing list setting, to create all new mails and replies in this folder with the corresponding mail address of this specific account in the "From" field and with the corresponding signature as well.
I don't remember why I set my own email address in the receiver pattern field of all personal folders many many years ago. Maybe an earlier version of YAM required this to activate the mailing list support.
I cleared the receiver pattern field. Now all is working fine.

**Comment by tboeckel** [comment]
**Originally on 2014-02-26 08:48:23 +0100**

___
Replying to AmigaSurfer:
> This is not a mailing list folder. It's a folder for personal mails from a specific account. I'm misusing the mailing list setting, to create all new mails and replies in this folder with the corresponding mail address of this specific account in the "From" field and with the corresponding signature as well.

But exactly that's the point. You abused a feature for a purpose it was never intended for. Furthermore you left the "Destination address" (Zieladresse) field empty and got exactly the desired result: an empty To: field.

To sum it up: there is no bug to be fixed here.
